app/controllers/YourBillController.scala

```diff
@@ -82,10 +82,13 @@ object YourBillController {
     val dueDate: LocalDate = chargeDueDate(List(ass))
     val startDate: LocalDate = parseLocalDate(ass.taxPeriodFrom.value)
     val endDate: LocalDate = parseLocalDate(ass.taxPeriodTo.value)
-    InvoicePeriod(monthNumber(startDate), startDate, endDate, dueDate)
+    InvoicePeriod(monthNumberInTaxYear(startDate), startDate, endDate, dueDate)
   }
 
-  def monthNumber(date: LocalDate): Int = (date.getMonth.getValue - 4) % 12
+  def monthNumberInTaxYear(date: LocalDate): Int = {
+    val month: Int = date.getMonth.getValue
+    if (month >= 4) month - 3 else month + 9
+  }
```

Comment:
So the date is not important? 
e.g. monthNumberInTaxYear(1 April) is same as monthNumberInTaxYear(10 April) ?

Reply:
Hmm that's a good point, it's for this bit 
<img width="733" alt="Screenshot 2022-08-31 at 15 19 37" src="https://user-images.githubusercontent.com/33868990/187701735-1df26b0d-a6fb-4f62-b892-56a2b55740f7.png">

Reply:
I'll add something for this, not sure if it will ever be a problem, but worth adding just in case